packages/dependency-extraction-webpack-plugin/test/__snapshots__/build.js.snap

```diff
@@ -92,7 +92,7 @@ exports[`Webpack \`no-default\` should produce expected output: Asset file 'main
 
 exports[`Webpack \`no-default\` should produce expected output: External modules should match snapshot 1`] = `Array []`;
 
-exports[`Webpack \`no-deps\` should produce expected output: Asset file 'main.asset.php' should match snapshot 1`] = `"<?php return array('dependencies' => array(), 'version' => 'd77fa1fabdfc18f9520a9325e0768b24');"`;
+exports[`Webpack \`no-deps\` should produce expected output: Asset file 'main.asset.php' should match snapshot 1`] = `"<?php return array('dependencies' => array(), 'version' => '18b79cbe0472a077b7cad085966aae12');"`;
 
```

Comment:
No sure what will happen here, it's a different version.

Reply:
It's all good because the content of the source file has changed:

```diff
- /* Silence is golden */
+ /* Silence is golden. */
```

I assume we don't use the minification for the generated test output so the calculated has has changed.